sdk/runanywhere-kotlin/src/jvmMain/kotlin/com/runanywhere/sdk/services/DeviceRegistrationServiceJvm.kt

```
@@ -0,0 +1,124 @@
+        try {
+            val rootPath = java.io.File("/")
+            if (rootPath.exists()) {
+                info["total_storage_mb"] = rootPath.totalSpace / (1024 * 1024)
+                info["available_storage_mb"] = rootPath.freeSpace / (1024 * 1024)
+            } else {
+                // Fallback for Windows
+                val userHome = java.io.File(System.getProperty("user.home"))
+                info["total_storage_mb"] = userHome.totalSpace / (1024 * 1024)
+                info["available_storage_mb"] = userHome.freeSpace / (1024 * 1024)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)